src/components/FreestarAdSlot/freestarWrapper.js

```diff
@@ -1,74 +1,81 @@
-
 class FreestarWrapper {
 
-  constructor() {
-    this.loaded = false;
-  }
-  get freestar() {
-    return this._freestar;
-  }
-
-  get googletag() {
-    return this._googletag;
+  constructor () {
+    this.loaded = false
   }
 
-  load(publisher) {
-    if (this.loaded === false){
-      this.loaded = true;
+  load (publisher) {
+    if (!this.loaded) {
+      this.loaded = true
       const qa = window.location.search.indexOf('fsdebug') > -1 ? '/qa' : ''
       const url = `https://a.pub.network/${publisher}${qa}/pubfig.min.js`
+
       window.freestar = window.freestar || {}
       window.freestar.hitTime = Date.now()
       window.freestar.queue =  window.freestar.queue || []
       window.freestar.config =  window.freestar.config || {}
-      window.freestar.config.enabled_slots = window.freestar.config.enabled_slots || [];
+      window.freestar.config.enabled_slots = window.freestar.config.enabled_slots || []
+
       const script = document.createElement('script')
       script.src = url
       script.async = true
       document.body.appendChild(script)
     }
-
   }
-  newAdSlot( placementName, onNewAdSlotsHook, channel, targeting) {
+
+  newAdSlot (placementName, onNewAdSlotsHook, channel, targeting) {
     window.freestar.queue.push(() => {
       window.freestar.newAdSlots({
         slotId: placementName,
         placementName,
         targeting
       }, channel)
-      if (onNewAdSlotsHook)
+      if (onNewAdSlotsHook) {
         onNewAdSlotsHook(placementName)
+      }
     })
 
   }
-  deleteAdSlot( placementName, onDeleteAdSlotsHook) {
+
+  deleteAdSlot (placementName, onDeleteAdSlotsHook) {
     window.freestar.queue.push(() => {
       window.freestar.deleteAdSlots({ placementName })
-      if ( onDeleteAdSlotsHook )
+      if (onDeleteAdSlotsHook) {
         onDeleteAdSlotsHook(placementName)
+      }
     })
   }
 
-  setPageTargeting( key, value) {
-    window.freestar = window.freestar || {};
+  refreshAdSlot (placementName, onAdRefreshHook) {
+    window.freestar.queue.push(() => {
+      window.freestar.newAdSlots()
```

Comment:
i believe there is a freestar.refreshAdSlot api to use here and need to pass in the placement name i believe ?